apps/settings/style/icons.css

```diff
@@ -16,7 +16,7 @@ a.menu-item:after {
   content: "forward-light";
   position: absolute;
   top: 0;
-  right: -0.9rem;
+  right: 0.3rem;
```

Comment:
Pavel is working on this inbug 1141840[1], so let's not make the change. https://bugzilla.mozilla.org/show_bug.cgi?id=1141840